andela/bp-andela-media
# #161551230 Continuous Integration

#### What does this PR do?
Adds TravisCI and CodeClimate for continous integration and deployment.
#### Description of Task to be completed?
Ensure CI and CD is setup properly.
#### How should this be manually tested?
Check the travis builds and codeclimate reports to see if the setup is done properly.
#### What are the relevant pivotal tracker stories?
[#161551230](https://www.pivotaltracker.com/story/show/161551230)
